src/lock/win32/lock.c

```diff
@@ -5,7 +5,9 @@
  */
 #undef  _WIN32_WINNT
 #define _WIN32_WINNT 0x0400
+#define WIN32_LEAN_AND_MEAN
 #include <windows.h>
+#undef WIN32_LEAN_AND_MEAN
```

Comment:
would it be better to move the WIN32_LEAN_AND_MEAN macro to the VS project file?